docs/fundamentals/code-analysis/quality-rules/ca1008.md

```diff
@@ -50,11 +50,21 @@ Do not suppress a warning from this rule except for flags-attributed enumeration
 Use the following option to configure which parts of your codebase to run this rule on.
 
 - [Include specific API surfaces](#include-specific-api-surfaces)
+- [Additional enum None names](#additional-enum-none-names)
 
 You can configure this option for just this rule, for all rules, or for all rules in this category (Design). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
 
 [!INCLUDE[api-surface](~/includes/code-analysis/api-surface.md)]
 
+### Additional enum None names
```

Comment:
I'm afraid `None` may get localized while it shouldn't (applies to all occurrences of it)

Not sure if this should have backticks around or a no-loc.

Reply:
Good remark, I don't know how this work for docs. @gewarren what's the guidance here please?